scylla/src/transport/session.rs

```diff
@@ -1653,7 +1653,10 @@ impl Session {
         QueryFut: Future<Output = Result<ResT, QueryError>>,
         ResT: AllowedRunQueryResTType,
     {
-        let mut last_error: Option<QueryError> = None;
+        // set default error as no known found as the query plan returns an empty iterator if there are no nodes in the plan
+        let mut last_error: Option<QueryError> = Some(QueryError::NoKnownNodeFoundError(
+            "Please confirm the supplied datacenters exists".to_string(),
+        ));
```

Comment:
I see we don't have access to provided datacenter name so we can't use it in error message. But the message could be better I think - right now it explains a probable fix, but does not say what went wrong.
Maybe something like `Load balancing policy returned empty query plan. It can happen when driver is provided with non-existing datacenter name`.